### DISCUSSION TITLE
iCloud Private Relay

### DISCUSSION THREAD

**Discussion by hectour3** [SelfControlApp/selfcontrol#691] [Q&A]
Can someone who has the MacOS Monterey beta running confirm whether Selfcontrol works with the new iCloud Private Relay feature on Safari?

**Comment by warek** [comment]
it creates conflicts. Private Relay won't work after you had SelfControl active. After restarting the computer (without a SelfControl timer running), it works again. 

If you have Privat Relay running and then start a SelfControl timer, SelfControl works, but then once the timer is over the WiFi stops working until you restart.
